composer.json

```diff
@@ -25,7 +25,8 @@
         "mockery/mockery": "^1.4",
         "orchestra/testbench": "^6.0",
         "phpunit/phpunit": "^9.4",
-        "spatie/valuestore": "^1.2"
+        "spatie/valuestore": "^1.2",
+        "laravel/octane": "^0.1.1"
```

Comment:
I guess this one should go when the PR gets merged?

Reply:
Yes, but it's only for dev purpose, because isn't required by the package.